web/js/map-OpenLayers.js

```diff
@@ -710,48 +697,29 @@ OpenLayers.Control.Click = OpenLayers.Class(OpenLayers.Control, {
         // Store pin location in form fields, and check coverage of point
         fixmystreet_update_pin(lonlat);
 
-        // Already did this first time map was clicked, so no need to do it again.
-        if (fixmystreet.page == 'new') {
-            return;
+        // If there are notes to be displayed, add the .with-notes class
+        // to make the sidebar wider.
+        if($('#report-a-problem-sidebar').length){
+          $('#map_box, #map_sidebar').addClass('with-notes');
```

Comment:
As this is only added here with a map click on around, this means that, with JS *on*, if the form is submitted and there's a server side error (e.g. bad password) and it returns the form for resubmission, the sidebar then moves inline (as it behaves without JS). I think this can be fixed server side - if we include the sidebar HTML early in the process when there might be a sidebar (so I think that's around and report/new), then it can know whether to put this class in.
As an aside, presumably the class could be added as a parent of both map_box and map_sidebar to save them both having to have it? Though if it's done server side, don't suppose that matters.